unstructured_api_tools/pipelines/templates/pipeline_api.txt

```diff
@@ -40,9 +40,12 @@ def get_validated_mimetype(file):
     if not content_type or content_type == "application/octet-stream":
         content_type = mimetypes.guess_type(str(file.filename))[0]
 
-        # Markdown mimetype is too new for the library - just hardcode that one in for now
-        if not content_type and ".md" in file.filename:
-            content_type = "text/markdown"
+        # Some filetypes missing for this library, just hardcode them for now
+        if not content_type:
+            if ".md" in file.filename:
```

Comment:
is this too lenient a condition compared with say endswith?

Reply:
yes, definitely!